samples/javascript/README.md

```diff
@@ -0,0 +1,129 @@
+
+## Start the DevContainer
+
+Go to the `samples/javascript` directory then open the Visual Studio Code.
+
+```
+$ cd samples/javascript
+$ code .
+```
+
+Visual Studio might automatically ask you to start container, if not, you can click the right bottom green icon (><) then you will see the following dropd down.
+
+![Remote Container](../../docs/images/RemoteContainer.png)
+
+Select `Remote-Containers: Reopen in Container`. It start the DevContainer, wait a couple of minutes, you will find a java development enviornment and a local kafka cluster is already up with Visual Studio Code.
+
+### Two Smaples
+
+This sample contains three functions. `Kafka Cluster` local means, it uses a kafka cluster that is started with DevContainer.
+
+| Name | Description | Kafka Cluster| Enabled |
+| ----- | --------------- | -------| ---|
+| UsersTrigger | Simple Kafka trigger sample | local | yes |
+| UsersTriggerMany | Kafka batch processing sample with Confluent Cloud | Confluent Cloud | no |
+
+
+### Modify function.json_ and local.settings.json
+
+If you want to use `UsersTriggerMany` sample, rename `UsersTriggerMany/function.json_` to `UsersTriggerMany/function.json` then Azure Functions Runtime will detect the function.
+
+Then copy `local.settings.json.example` to `local.settings.json` and configure your ccloud environment.
+
+### Modify UsersTriggerMany/function.json (Windows user only)
+
+If you want to run sample on your Windows with Confluent Cloud and you are not using DevContainer, uncomment the following line. It is the settings of CA certificate. .NET Core, that is azure functions host language, can not access windows registry, that means can not access the CA certificate of the Confluent Cloud.
+
+_UserTriggerMany/function.json_
+
+```json
+"sslCaLocation":"confluent_cloud_cacert.pem",
+```
+
+For downloading `confluent_cloud_cacert.pem`, you can refer to [Connecting to Confluent Cloud in Azure](https://github.com/Azure/azure-functions-kafka-extension#connecting-to-confluent-cloud-in-azure).
+
+## Install the KafkaTriggerExtension
+
+This command will install Kafka Extension. The command refer the `extensions.csproj` then find the Kafka Extension NuGet package.
+
+```bash
+$ func extensions install
+```
+
+Check if there is dll packages under the `target/azure-functions/kafka-function-(some number)/bin`. If it is sucess, you will find `Microsoft.Azure.WebJobs.Extensions.Kafka.dll` on it. 
+
+## Run the Azure Functions
+
+## Run 
+
+Before running the kafka extension, you need to configure `LD_LIBRARY_PATH` to the `/workspace/bin/runtimes/linux-x64/native"`. For the DevContainer, the configuration is reside in the `devontainer.json`. You don't need to configure it.
+
+```bash
+$ func start
+```
+
+### deploy to Azure
+
+#### deploy the app
+
+Deploy the app to a Premium Function You can choose.
```

Comment:
Yes. We support Premium. not consumption. Since we need to provide scale controller. This repo includes scale controller for Premium. Thank you for sharing the document. I add the link. Thank you!